example_bytecode_test.go

```diff
@@ -1,4 +1,4 @@
-package duktape_test
+package duktape
```

Comment:
I don't mind, but then you need to setup Wercker CI initialize script to download dependencies (e.g. to execute command "go get gopkg.in/olebedev/go-duktape.v3") or it would be broken.